ansible_collections/arista/avd/plugins/plugin_utils/utils/load_python_class.py

```diff
@@ -0,0 +1,44 @@
+import importlib
+from typing import Optional
+
+from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
+
+
+def load_python_class(module_path: str, class_name: str, parent_class: Optional[type] = None) -> type:
```

Comment:
We don't use Optional anywhere else. I'm not sure we need it here. Could be `: type | None`

Reply:
https://mypy.readthedocs.io/en/stable/command_line.html#none-and-optional-handling

https://peps.python.org/pep-0484/#union-types

_This is no longer the recommended behavior. Type checkers should move towards requiring the optional type to be made explicit._